c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -3300,6 +3312,9 @@ namespace CompressionStream {
   bool constructor(JSContext* cx, unsigned argc, Value* vp) {
     // 1.  If _format_ is unsupported in `CompressionStream`, then throw a `TypeError`.
     CallArgs args = CallArgsFromVp(argc, vp);
+    if (!ThrowIfNotConstructing(cx, args, "CompressionStream")) {
+      return false;
+    }
     if (!args.requireAtLeast(cx, "CompressionStream constructor", 1)) {
       return false;
     }
```

Comment:
... and this would become
```suggestion
    CTOR_HEADER("CompressionStream", 1)
```